src/status_im/ui/screens/wallet/send/views.cljs

```diff
@@ -84,22 +84,22 @@
 
 ;; "Sign Later" and "Sign Transaction >" buttons
 (defn- sign-buttons [amount-error to amount sufficient-funds? sign-later-handler modal?]
```

Comment:
@ConnorChristie could you please elaborate how changes in `sign-buttons` function related to this PR? thanks

Reply:
Yes, it was a bug that occurred when an insufficient funds error was present after adding the logic to support tokens. The previous code showed the sign later button only when `sign-enabled?` was true but it should be the sign transaction button that is shown when signing is enabled after validation

Reply:
Merged and fixed conflicts!